python/ray/rllib/optimizers/async_replay_optimizer.py

```diff
@@ -334,8 +394,8 @@ def step(self):
                 grad_out = self.local_evaluator.compute_apply(replay)
                 for pid, info in grad_out.items():
                     prio_dict[pid] = (
-                        replay.policy_batches[pid]["batch_indexes"],
-                        info["td_error"])
+                        replay.policy_batches[pid].data.get("batch_indexes"),
+                        info.get("td_error"))
```

Comment:
do you need to set defaults or will None be fine

Reply:
default is None